eoscostarica/guias.eoscostarica.io
# Add content in open source projects page

### Open source projects page

### What does this PR do?

- resolves #315

### Steps to test
1. Go to "Pautas para desarrolladores"
2. Open "Proyectos de Código Abierto"
3. Check the content and the added projects: myvote eos dao, eden smart contracts, constancias municipales, eos starter token sale contract and token sale platform

#### CheckList
